arcgis_map_sdk_ios/ios/Classes/ParseException.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  ParseException.swift
+//  arcgis_map_sdk_ios
+//
+//  Created by Julian Bissekkou on 26.11.23.
+//
+
+import Foundation
+
+
+struct ParseException: Error {
+    let message: String
+}
```

Comment:
When checking the branch locally on your maschine, do you see this file in xcode? 
Works on my maschine, but I would expect a change in the xcode project file that references to this new file...